src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,86 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
+  amount: string
+  symbol: string
+  discountPercent: string
+  isClaimed?: boolean
+  onClaim?: () => void
+}
+
+const actionsPaddingLeft = { base: 10, md: 0 }
+const columnsWidth = { base: '100%', md: '50%' }
+const columnsDirection: StackDirection = { base: 'column', md: 'row' }
+const columnsAlignItems = { md: 'center' }
+const columnsJustifyContent = { md: 'space-between' }
+const columnsSpacing = { base: 4, md: 48 }
+
+export const FoxWifHatHoldingLine = ({
+  accountId,
+  accountNumber,
+  amount,
+  symbol,
+  discountPercent,
+  isClaimed,
+  onClaim,
+}: FoxWifHatHoldingLineProps) => {
+  const textColor = useColorModeValue('gray.500', 'gray.400')
+  const translate = useTranslate()
+
+  return (
+    <Stack
+      width='full'
+      px={6}
+      py={4}
+      spacing={columnsSpacing}
+      direction={columnsDirection}
+      alignItems={columnsAlignItems}
+      justifyContent={columnsJustifyContent}
+    >
+      <Flex width={columnsWidth} alignItems='center' justifyContent='space-between'>
+        <HStack spacing={4}>
+          <WalletIcon color={textColor} boxSize={6} />
+          <Flex direction='column' alignItems='flex-start'>
+            <Text fontWeight='bold' fontSize='sm'>
+              {middleEllipsis(accountId)}
+            </Text>
+            <Text color={textColor} fontSize='xs'>
+              {translate('accounts.accountNumber', { accountNumber })}
+            </Text>
+          </Flex>
+        </HStack>
+
+        <Text fontWeight='bold'>
+          {amount} {symbol}
+        </Text>
+      </Flex>
+
+      <Flex
+        width={columnsWidth}
+        alignItems='center'
+        justifyContent='space-between'
+        pl={actionsPaddingLeft}
+      >
+        <Text color='green.500' fontSize='sm' fontWeight='bold'>
+          {translate('foxPage.foxWifHat.discountText', { percent: discountPercent })}
+        </Text>
+
+        <Button
+          colorScheme={isClaimed ? 'green' : 'gray'}
+          size='sm'
+          onClick={!isClaimed ? onClaim : undefined}
+        >
```

Comment:
We'll want to disable the button/hide the hover state if already claimed.

Reply:
It was wanted to respect the mock but yeah, trying to be pragmatic and disabling it feels better! 👍 

Done in [2e3a203](https://github.com/shapeshift/web/pull/8605/commits/2e3a203909b4eca964e0f8520208a6f5cc7c099c)